src/njoy21/input/PLOTR/src/readCard13List.hpp

```diff
@@ -0,0 +1,14 @@
+template< typename Istream>
+static optional< std::vector< Card13 > > 
+                                 readCard13List( Istream& is, const int nform ){
+  if( nform != 0 ) return std::nullopt;
+
+  std::vector< Card13 > card13List;
```

Comment:
@apmccartney will suggest this needs to be the first line of the method to enable the compiler to perform Return Value Optimization.